config/runtime.exs

```diff
@@ -18,30 +26,36 @@ source!([
 ])
 
 # Start the phoenix server if environment is set and running in a release
-if System.get_env("PHX_SERVER") && System.get_env("RELEASE_NAME") do
+if env!("PHX_SERVER", :boolean, false) && env!("RELEASE_NAME", :boolean, false) do
   config :lightning, LightningWeb.Endpoint, server: true
 end
 
-if is_resettable_demo = System.get_env("IS_RESETTABLE_DEMO") do
-  config :lightning,
-         :is_resettable_demo,
-         is_resettable_demo == "yes"
-end
+config :lightning,
+       :is_resettable_demo,
+       env!(
+         "IS_RESETTABLE_DEMO",
+         :boolean,
```

Comment:
.env.example needs to be updated with `IS_RESETTABLE_DEMO=false` because IS_RESETTABLE_DEMO=no is evaluated to true:
```
$ export IS_RESET=no; iex
iex(1)> Mix.install [{:dotenvy, "0.8.0"}]
:ok
iex(2)> Dotenvy.env!("IS_RESET", :boolean, false)
true
```
or force the use of with `:boolean!`

Reply:
Good catch! I've changed it to use `ensure_boolean/1` which accepts `true|false|yes|no` - which also rejects other arbitrary values and don't consider the existance of the var as `true`.